src/hu/ssh/github_changelog/dependencies/bundler.clj

```diff
@@ -0,0 +1,22 @@
+(ns hu.ssh.github-changelog.dependencies.bundler
+  (:require
+    [clojure.string :refer [split-lines]]))
+
+(defn- get-specs [file]
+  (->> (split-lines file)
+       (drop-while #(not= % "  specs:"))
+       (drop 1)
+       (take-while seq)))
+
+(defn- parse-spec [spec]
+  {:name (second spec) :version (nth spec 2)})
+
+(defn- parse-specs [specs]
+  (->> (map #(re-matches #"^\s{4}(\S+) \((.*)\)$" %) specs)
+       (remove empty?)
+       (map parse-spec)))
+
+(defn parse [file]
+  (->> (slurp file)
```

Comment:
Why is that better? Lazyness is not required here, since we need all the files and with `doall` it just makes everything a bit more complicated.